core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -916,7 +926,7 @@ class KafkaController(val config : KafkaConfig, zkUtils: ZkUtils, val brokerStat
             // Resign if the controller is in an illegal state
             error("Forcing the controller to resign")
             brokerRequestBatch.clear()
-            controllerElector.resign()
+            controllerElector.resign
```

Comment:
is this an inconsistent use of parens between `resign` and `initializeControllerElector()`?